CardForm/src/main/res/layout/bt_card_form_fields.xml

```diff
@@ -71,6 +71,28 @@
 
     <LinearLayout
         style="@style/bt_field_container"
+        tools:ignore="RtlCompat"
```

Comment:
Personally, I have entered my credit card number bunch of times in the name field, later realizing there was another credit card number field in view.